sample_factory/algo/learning/learner.py

```diff
@@ -758,7 +758,7 @@ def _train(
 
                     kl_old_mean = float(kl_old.mean().item())
                     recent_kls.append(kl_old_mean)
-                    if kl_old.max().item() > 100:
+                    if kl_old.numel() > 0 and kl_old.max().item() > 100:
```

Comment:
This will fix the crash but I still don't understand how we got a 0-dim tensor here...